Formula/analog.rb

```diff
@@ -26,11 +26,19 @@ class Analog < Formula
   uses_from_macos "zlib"
 
   def install
-    system "make", "CC=#{ENV.cc}",
-                   "CFLAGS=#{ENV.cflags}",
-                   "DEFS='-DLANGDIR=\"#{pkgshare}/lang/\"' -DHAVE_ZLIB",
-                   "LIBS=-lz",
-                   "OS=OSX"
+    args = %W[
+      CC=#{ENV.cc}
+      CFLAGS=#{ENV.cflags}
+      DEFS='-DLANGDIR=\"#{pkgshare}/lang/\"'\ -DHAVE_ZLIB
```

Comment:
I think you don't need to escape double-quotes here, actually.

Reply:
Right. That was legacy code. I had to escape the space though.